docs/yaml_config.rst

```diff
@@ -0,0 +1,338 @@
+YAML Configuration
+==================
+
+*SOSS* is configured by means of a YAML file that specifies a set of compulsory fields, plus some optional ones.
+The most common fields required to configure a **System-Handle** are:
+
+* :code:`types`: specifies the IDL types used by *SOSS* to transmit messages
+* :code:`systems`: specifies the middlewares involved in the communication
+* :code:`routes`: specifies which bridges SOSS needs to create
+* :code:`topics`/:code:`services`: specify the topics exchanged over the above bridges in either publisher/subscriber
+  or client/server type communications
+
+A generic YAML communicating two systems has the following structure:
+
+.. code-block:: yaml
+
+    types:
+        idl: <idl_content>
+        paths: [idl_include_path_1, id_include_path_2 ]
+    systems:
+        <system_1_name>: { type: <system_1_type>, types-from: <other_system_name>, <system_1_config> }
+        <system_2_name>: { type: <system_2_type>, types-from: <other_system_name>, <system_2_config> }
```

Comment:
I would restructure these code blocks into a more friendly format for the reader; also, in the resulting output, if your browser window is not big enough, you will end up scrolling back and forth because the lines are too long. Something like:
```
systems:
    <system_1_name>: {
        type: <system_1_type>,
        types-from: <other_system_name>, <system_1_config>
    }
    ...
```
works better for me, and so on with the rest of the YAML code blocks.

Reply:
@LuisGP

Reply:
Updated.